build.gradle

```diff
@@ -86,10 +86,6 @@ ext.additionalPomInfo = {
 }
 
 subprojects {
-    // required plugins
-    apply plugin: "base"
```

Comment:
Does this mean that the message about local build detected, now also only shows once and not for every subproject?

Reply:
I don’t think so, that’s future work.